cmd/configure.go

```diff
@@ -26,7 +26,7 @@ var defaultColors = map[string]string{
 // the colors of the text output
 var configureColorsCmd = &cobra.Command{
 	Use:   "colors",
```

Comment:
```suggestion
	Use:   "colors [--color <value>]...",
```

for consistency? i dunno

Reply:
I think this one is different because colors are flags so it's technically included in `[flags]` that shows up in the help.

![image](https://github.com/user-attachments/assets/4692d57f-bf34-47e7-a841-80136dbc06bc)

Reply:
ah cool makes sense to me